gobblin-runtime/src/main/java/gobblin/runtime/LimiterFactory.java

```diff
@@ -0,0 +1,87 @@
+/* (c) 2014 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.runtime;
+
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.base.Preconditions;
+
+import gobblin.configuration.ConfigurationKeys;
+import gobblin.configuration.State;
+
+
+/**
+ * A factory class for {@link Limiter}s.
+ *
+ * @author ynli
+ */
+public class LimiterFactory {
```

Comment:
nit: can you rename to StandardLimiterFactory or something along those lines. LimiterFactory sounds a bit too generic. Someone else may want to implement their own limiter and limiter factory.